tests/Avalonia.Markup.UnitTests/Parsers/ExpressionObserverBuilderTests_Negation.cs

```diff
@@ -168,10 +168,23 @@ public void SetValue_Should_Return_False_For_Invalid_Value()
 
         private class Test : INotifyDataErrorInfo
         {
+            private string _dataValidationError;
+
             public bool Foo { get; set; }
             public object Bar { get; set; }
 
-            public string DataValidationError { get; set; }
+            public string DataValidationError
```

Comment:
Why was this change needed?

Reply:
my mistake

Reply:
CS0067: The event 'event' is never used.